src/cli/README_SRP.md

```diff
@@ -0,0 +1,26 @@
+# OpenThread CLI - SRP (Service Registration Protocol)
+
+## Quick Start
+
+### Build with SRP Client support
+
+Use the `SRP_CLIENT=1 ECDSA=1` build switches to enable SRP Client API support (which requires ECDSA support).
+
+```bash
+> ./bootstrap
+> make -f examples/Makefile-simulation SRP_CLIENT=1 ECDSA=1
+```
+
+## Example
+
+Print supported modes, `client`.
+
+```bash
+> srp
+client
```

Comment:
Should this be on one line like in README_SRP_CLIENT.md?

Reply:
The idea is that if we use `srp` command on its own it (with no command/args), it would print the list of its sub-commands which for now  is only `client` but we can later have `server` as well.